.travis.yml

```diff
@@ -18,95 +18,95 @@ before_install:
 after_success:
   - docker --version
 
-matrix:
+jobs:
   include:
-    - name: "Build Frappe bench development environment (latest)"
+    - stage: "Build Frappe bench development environment (latest)"
       if: branch = develop AND type != pull_request
       script:
         - docker build -t frappe/bench:latest -f build/bench/Dockerfile .
         - docker push frappe/bench:latest
-    - name: "Build Frappe python environment (edge)"
+    - stage: "Frappe (edge)"
       if: branch = develop AND type != pull_request
       script:
         - ./travis.py frappe --nginx --tag edge
         - ./travis.py frappe --nginx --tag develop--tag-only
-    - name: "Build Frappe nginx + static assets (edge)"
+    - stage: "Frappe (edge)"
```

Comment:
I'm confused?

Is this repeated?

/travis.py frappe --nginx --tag edge

Reply:
We need to use stages so that the `frappe-*` images are built first, otherwise, we'll get the `install_app not found` error.

But each stage will run separately making builds take much longer, so if we name the stages the same thing, travis-ci will run them together.

Reply:
I think the frappe-nginx is built twice?

line 28 to line 37 .travis.yml

```
    - stage: "Frappe (edge)"
      if: branch = develop AND type != pull_request
      script:
        - ./travis.py frappe --nginx --tag edge
        - ./travis.py frappe --nginx --tag develop--tag-only
    - stage: "Frappe (edge)"
      if: branch = develop AND type != pull_request
      script:
        - ./travis.py frappe --nginx --tag edge
        - ./travis.py frappe --nginx --tag develop--tag-only
```